src/util/helperfunctions.ts

```diff
@@ -21,4 +21,8 @@ export function handleSpecialStickerNames(name: string): string {
         return name.substring(0, name.lastIndexOf('|')) + ' ' + name.substring(name.lastIndexOf('|'), name.length);
     }
     return name;
+}
+
+export function calculateRelativePercentageDifference(a: number, b: number) {
```

Comment:
This function is currently not in use, correct? I would love to see it removed then.

Reply:
> This function is currently not in use, correct? I would love to see it removed then.

oh my bad, it was being used before and i got rid of it in the main function but forgot to delete from the helperfunctions

Reply:
done, removed the function